dspace-spring-rest/src/main/java/org/dspace/app/rest/AuthenticationRestController.java

```diff
@@ -111,6 +133,39 @@ public ResponseEntity logout() {
         return ResponseEntity.noContent().build();
     }
 
+    @RequestMapping(value = "/patch", method = {RequestMethod.PATCH})
```

Comment:
we don't need to offer a separate endpoint for patching done by the current user on his profile. It should be managed transparently by the REST implementation on the current endpoint

Reply:
Hi @abollini 
Sorry for the delay in getting back to this. I did have some doubts about the new endpoint and happy to take a look at the PR provided by @tomdesair. I believe I understand what you are suggesting here, so shouldn't take long to update this PR.